src/project/manifest_registry/member_config.rs

```diff
@@ -23,7 +23,7 @@ impl MemberConfig {
 }
 
 fn merge_serde_json_value(a: &mut Value, b: &Value) {
-    if let (Value::Object(ref mut a_map), Value::Object(ref b_map)) = (a, b) {
+    if let (Value::Object(a_map), Value::Object(b_map)) = (a, b) {
```

Comment:
is this related tho?

Reply:
this is edition change: https://blog.rust-lang.org/2025/02/20/Rust-1.85.0.html